src/components/search/search.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { SearchBar } from "@nypl/design-system-react-components";
+import { useRouter } from "next/router";
+import PublicDomainFilter from "../publicDomainFilter/publicDomainFilter";
+
+const Search = () => {
+  const router = useRouter();
+  const [keywords, setKeywords] = useState("");
+  const [publicDomainOnly, setPublicDomainOnly] = useState(false);
+
+  const handleSubmit = (event, router) => {
+    event.preventDefault();
+    const searchUrl =
+      `https://digitalcollections.nypl.org/search/index?` +
```

Comment:
The prefix should be `appConfig.DC_URL`. QA will be testing this expecting qa-dcfrontend.nypl.org to search qa-digitalcollections.nypl.org and production dcfrontend.nypl.org to search production digitalcollections.nypl.org.